README.md

```diff
@@ -237,7 +237,7 @@ volumes:
 services:
   stonework:
     container_name: stonework
-    image: "ghcr.io/pantheontech/stonework"
+    image: "ghcr.io/pantheontech/stonework:21.06.0"
```

Comment:
The tag should actually be `21.06` - rolling release which includes patch releases as well.